src/components/admin/personen/PersonLock.vue

```diff
@@ -43,12 +86,27 @@
     return schulen.value.length <= 1;
   });
 
+  function resetBefristungFields(): void {
+    isUnbefristet.value = true;
+    formContext.resetField('selectedBefristung');
+    selectedBefristungProps.value.error = false;
+    selectedBefristungProps.value['error-messages'] = [];
+  }
+
   function closeLockPersonDialog(isActive: Ref<boolean>): void {
+    resetBefristungFields();
     isActive.value = false;
   }
-  function handleOnLockUser(id: string, isActive: Ref<boolean>): void {
+  async function handleOnLockUser(id: string, isActive: Ref<boolean>): Promise<void> {
+    if (!isUnbefristet.value) {
+      let { valid }: FormValidationResult<PersonLockForm, PersonLockForm> = await formContext.validate();
+      if (!valid) return;
+    }
```

Comment:
Wenn du die Validierung nur triggern möchtest, wenn isUnbefristet false ist, dann bitte das in die ValidationSchema machen. Das geht schnell mit "when().is " ... gucke dir mal z.B. das in PersonCreationView.

Reply:
done